packages/storybook-test/stories/icon/icon.stories.tsx

```diff
@@ -0,0 +1,132 @@
+      </svg>
+    ),
+  },
+  argTypes: {
+    'aria-label': { table: { category: 'API' }, type: 'string' },
+    'aria-labelledby': { table: { category: 'API' }, type: 'string' },
+    // Hide children as it's a React component which cannot be displayed nicely in the Storybook UI
+    children: { table: { disable: true } },
+    className: { table: { category: 'API' }, type: 'string' },
+    dir: {
+      control: { labels: { undefined: '(undefined)' }, type: 'select' },
+      options: [undefined, 'rtl', 'ltr', 'auto'],
+      table: { category: 'API' },
+    },
+    role: {
+      control: { labels: { undefined: '(undefined)' }, type: 'select' },
+      options: [undefined, 'img'],
+      table: { category: 'API' },
+    },
+  },
+  component: Icon,
+  parameters: {
+    externalLinks: [
+      {
+        name: 'Open in Figma',
+        url: 'https://www.figma.com/design/shhwGcqPLi2CapK0P1zz8O/NLDS---Voorbeeld---Bibliotheek?node-id=153-1056',
+      },
+      {
+        name: 'Open op GitHub',
+        url: packageJSON.homepage,
+      },
+    ],
+  },
+  title: 'Componenten/Icon',
+} satisfies Meta<typeof Icon>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const AriaHidden: Story = {
+  name: 'Decoratief icoon met aria-hidden',
+  parameters: {
+    docs: {
+      description: {
+        story:
+          'Wanneer een icon decoratief is, is het niet nodig dat een bezoeker die gebruik maakt van hulpsoftware weet dat er een icon aanwezig is. Daarom is het nodig dat de icon aria-hidden ondersteunt.',
+      },
+    },
+  },
+};
+
+export const RoleImg: Story = {
+  name: 'Icoon met afbeelding rol',
+  args: {
+    role: 'img',
+    'aria-label': 'Icoon',
+  },
+  parameters: {
+    docs: {
+      description: {
+        story:
+          'Wanneer een icon niet verborgen is voor hulpsoftware, moet de rol goed gecommuniceerd worden in de accessibility tree. Voor een icon is dit role="img".',
+      },
+    },
+  },
+};
+
+export const AriaLabel: Story = {
+  name: 'Icoon met aria-label',
+  args: {
+    role: 'img',
+    'aria-label': 'Icoon',
+  },
+  parameters: {
+    docs: {
+      description: {
+        story:
+          'Om ervoor te zorgen dat de icon een alternatieve tekst kan hebben, is het nodig dat het een `aria-label` attribuut kan bevatten.',
+      },
+    },
+  },
+};
+
+export const AriaLabelledby: Story = {
+  name: 'Icoon met aria-labelledby',
+  args: {
+    role: 'img',
+    'aria-labelledby': 'paragraph',
+  },
+  parameters: {
+    docs: {
+      description: {
+        story:
+          'Om ervoor te zorgen dat de icon een alternatieve tekst kan hebben, is het nodig dat het een `aria-labelledby` attribuut kan bevatten.',
+      },
+    },
+  },
+  render(props) {
+    return (
+      <>
+        <Paragraph id="paragraph">Een schitterend icoon</Paragraph>
+        <Paragraph>
+          <Icon {...props} />
+        </Paragraph>
+      </>
+    );
+  },
+};
```

Comment:
Zullen we nog rtl stories toevoegen voor `<Icon dir="rtl">` en `<Paragraph dir="rtl">`?